### ISSUE TITLE
[Grid] Add a keyboard navigation shortcut for grouping

### ISSUE THREAD

**Issue by WissamProgress** [telerik/kendo-react#1800]
## I'm submitting a...

  * Feature request

## Current behavior
Currently, the Grid component does not have built-in keyboard navigation for column grouping.
Here is an example where grouping is enabled - https://www.telerik.com/kendo-react-ui/components/grid/grouping/

## Expected behavior
The grouping feature should be keyboard accessible instead of having to drag and drop using the mouse, like in Kendo UI for jQuery, where it can be done using `ctrl+space` - https://demos.telerik.com/kendo-ui/grid/keyboard-navigation

Reported in Ticket ID: 1634185

**Comment by iadnonov** [comment]
Fixed in dev version [7.3.0](https://www.telerik.com/kendo-react-ui-develop/components/changelogs/ui-for-react/#v7.3.0)
